### ISSUE TITLE
SublimeCodeIntel does not seem to recurse in to sub directories

### ISSUE THREAD

**Issue by kerrsmith** [SublimeCodeIntel/SublimeCodeIntel#509]
I have just installed SublimeCodeIntel in to Sublime Text 3 on Windows 7 64bit and have been testing it out on my PHP code. If the files are all in the same directory I can Alt+Click and jump to a definition, if the relevant  file is not open it is opened up straight away.

If I try the same thing but the definition function is in a file in a sub folder then it does not open the file as it does when they are all in the same directory.

I have looked online and seen it does take some time to index the files so I left it for 15 minutes and it still did not work - if I move to the sub directory and try Alt+Click on a function with the definition in a file in that same directory it works instantly and takes me to the correct file as before.

In the preferences it show recursion is on but it does not seem to be working for me on subdirectories.

Thanks in advance for your help.

**Comment by Kronuz** [comment]
What is your directory structure? You must add the topmost parent directorie(s) to your Sublime Text project `Project -> Add folder to project` and that will include the files inside those directories in the index.

**Comment by kerrsmith** [comment]
That's great! It is now working as expected - I did not know I needed to add the main root directory to a project.

It might be worth mentioning this in the 'using' area of the help page - I have read through the instructions but did not see this mentioned.

Thanks once again for your help, I really appreciate it - this is a really great plugin and has already been extremely useful and I have only had it two days.
